files/default/supervisor-ckan-harvest-gather.conf

```diff
@@ -0,0 +1,47 @@
+[unix_http_server]
+file=/var/tmp/supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+; ====================================================================
+; Supervisor configuration for CKAN Harvest background gather consumer
+; ====================================================================
+
+; 1. Copy this file to /etc/supervisor/conf.d
+; 2. Make sure the paths below match your setup
+
+
+[program:ckan-harvest-gather]
+
+; Use the full paths to the virtualenv and your configuration file here.
+command=/usr/lib/ckan/default/bin/paster --plugin=ckan-harvest harvester gather_consumer --config=/etc/ckan/default/production.ini
+
+
+; User the gather consumer runs as.
+user=ckan
+
+
+; Start just a single gather consumer.
+numprocs=1
+process_name=%(program_name)s-%(process_num)02d
+
+
+; Log files.
+stdout_logfile=/var/log/ckan/ckan-harvest-gather.log
+stderr_logfile=/var/log/ckan/ckan-harvest-gather.log
```

Comment:
add note that these needs to by copied to s3 and setup to cloudwatch logs inside opsworks

Reply:
They should be picked up by the existing processes. See https://github.com/qld-gov-au/opswx-ckan-cookbook/blob/master/files/default/ckan-logrotate which rotates `/var/log/ckan/*log`